airflow/www/views.py

```diff
@@ -2788,11 +2895,22 @@ def tries(self, session=None):
         ]
     )
     @action_logging
+    def legacy_landing_times(self):
+        """Redirect from url param."""
+        return redirect(url_for('Airflow.landing_times', **request.args))
+
+    @expose('/dags/<string:dag_id>/landing_times')
```

Comment:
While we're changing this, could we use a hyphen instead? Hyphens are a more common, modern practice and we've used them in recent additions (e.g. `/rendered-k8s`, `/rendered-templates`).
```suggestion
    @expose('/dags/<string:dag_id>/landing-times')
```

Reply:
Done!